### DISCUSSION TITLE
Verify your firebase project

### DISCUSSION THREAD

**Discussion by Ken-JIAnomads** [AntlerVC/firetable#1207] [Support / Q&A] [answered]
Hi,

I'm during the set-up phase.  I set up a firebase project and still get the following message when I tried to verify "Looks like you still don't have a Firebase project setup. Please click the "Open Firebase console" button above and create a project on Firebase and come back here to click the "Verify" button after?

I checked accounts and it is taking me to the same account on Firebase, as in Rowy.

Any ideas why?

Thanks in advance. 
Ken

![Screenshot 2023-04-06 at 4 02 39 pm](https://user-images.githubusercontent.com/130031825/230285791-2c4a5cac-33b4-4c6a-a88c-7efd3cbaffb8.jpg)

**Comment by BeeBombshell** [comment, marked as answer]
Hey @Ken-JIAnomads, I see you closed this discussion as resolved. Just curious about how you resolved this... Did you accidentally create the project on a different account?

**Comment by Ken-JIAnomads** [comment]
Hi,

I logged out of Rowy and back in (via Google) and it resolved the conflict.

Thanks!

**Comment by lostprojects** [comment]
I'm actually struggling with the same issue. 

The Google accounts are definitely the same across Rowy and Firebase.

I've tried @Ken-JIAnomads method of logging out and back in on Rowy, and still the same error as experienced by Ken initially.
